scripts/db_migrator.py

```diff
@@ -178,24 +178,33 @@ def migrate_intf_table(self):
         if self.appDB is None:
             return
 
-        data = self.appDB.keys(self.appDB.APPL_DB, "INTF_TABLE:*")
-
-        if data is None:
+        # Get Lo interface corresponding to IP(v4/v6) address from CONFIG_DB.
+        configdb_data = self.configDB.get_keys('LOOPBACK_INTERFACE')
+        lo_addr_to_int = dict()
+        for int_data in configdb_data:
+            if type(int_data) == tuple and len(int_data) > 1:
+                intf_name = int_data[0]
+                intf_addr = int_data[1]
+                lo_addr_to_int.update({intf_addr: intf_name})
+
+        lo_data = self.appDB.keys(self.appDB.APPL_DB, "INTF_TABLE:lo*")
+        if lo_data is None:
             return
+        for lo_row in lo_data:
+            # Example of lo_row: 'INTF_TABLE:lo:10.1.0.32/32'
+            # Example of lo_dict: {'family': 'IPv4', 'scope': 'global'}
+            lo_dict = self.appDB.get_all(self.appDB.APPL_DB, lo_row)
+            lo_addr = lo_row.split('INTF_TABLE:lo:')[1]
+            lo_name = lo_addr_to_int.get(lo_addr)
```

Comment:
A logical difference from previous implementation is that, if config_db data is empty, previously we hardcode to 'Loopback0' whereas here it will come as empty which can cause swss crash. For safe side, I suggest if 'lo_name' is empty use the previous logic.

Reply:
Sure, addressed this in latest.